examples/allocation-endpoint/Makefile

```diff
@@ -30,7 +30,7 @@ PROD_REPO ?= us-docker.pkg.dev/agones-images/examples
 mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 project_path := $(dir $(mkfile_path))
 root_path = $(realpath $(project_path)/../..)
-version := 0.8
+version := 0.8  # Change this to ignore the version change presubmit: `check-makefile-version-increment`
```

Comment:
"ignore" ?

Not quite sure what this means. Or are you saying "if CI fails on `check-makefile-version-increment`` make sure to increment this value" ?

Reply:
I've updated the comment to say, "Bump this version if CI fails on `check-makefile-version-increment`"